app/src/main/java/com/dscvit/handly/model/collection/ViewCollectionsResponse.kt

```diff
@@ -0,0 +1,4 @@
+package com.dscvit.handly.model.collection
+
+
+class ViewCollectionsResponse : ArrayList<Collection>()
```

Comment:
We don't need to subclass for a response if we aren't adding anything extra.

Reply:
Yeah will fix this